pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -49,6 +46,9 @@ const (
 	nodeStatusOperandFailedFallbackReason = "OperandFailedFallback"
 )
 
+//go:embed manifests/installer-pod.yaml
+var template []byte
```

Comment:
`template` => `podTemplate`, what do you think?

Reply:
+1, done.